### PR TITLE
[4.0] Cleanup the View file com_cache

### PR DESCRIPTION
**Pull request by roland-d**
### Summary of Changes
This is the next step in cleaning up the view files and adhering to coding standards as well as making the views more usable by the IDE by removing the magic getter.

### Testing Instructions

1. Go to System -> Clear Cache
2. Make sure Cache page loads as before

### Documentation Changes Required
None

### CODE REVIEW

File: administrator/components/com_cache/tmpl/cache/default.php [hunk 1]
```diff
@@ -64,11 +67,11 @@
 									<?php echo '&#x200E;' . HTMLHelper::_('number.bytes', $item->size); ?>
 								</td>
 							</tr>
-						<?php $i++; endforeach; ?>
+						<?php endforeach; ?>
```
Comment (Quy) [inline administrator/components/com_cache/tmpl/cache/default.php]:
```suggestion
						<?php $i++; endforeach; ?>
```

Comment (Quy) [reply administrator/components/com_cache/tmpl/cache/default.php]: Needs to increment for lines 56 and 59.

File: administrator/components/com_cache/tmpl/cache/default.php [hunk 2]
```diff
@@ -14,6 +14,9 @@
 use Joomla\CMS\Layout\LayoutHelper;
 use Joomla\CMS\Router\Route;
 
+/** @var \Joomla\Component\Cache\Administrator\View\Cache\HtmlView $this */
+
```
Comment (Quy) [inline administrator/components/com_cache/tmpl/cache/default.php:18]:
```suggestion
```

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**